straxen/scada.py

```diff
@@ -237,7 +237,7 @@ def _query_single_parameter(self,
         if query_type_lab:
```

Comment:
Can we remove `query_type_lab` from this function, looks like a relic that is not used at all and is also still lingering in the docstring

Reply:
See my comment above. I have to discuss with SC-Team why it is not working yet.